src/main/java/com/openshift/internal/restclient/model/ReplicationController.java

```diff
@@ -364,4 +366,24 @@ public void addVolume(IVolumeSource volumeSource) {
 		}
 		volList.add(ModelNode.fromJSONString(volumeSource.toJSONString()));
 	}
+
+    @Override
+    public void setServiceAccount(String serviceAccount) {
```

Comment:
RC does not implement Pod because it is not a Pod; we need to keep some relative relationship between the client and server code.  What I am saying is for RC there is no entry in the json tree for 'serviceaccount'.  There is no need for this method to exist as far as i can see as it will not provide any useful information to the controllers on the server: https://github.com/openshift/origin/blob/master/vendor/k8s.io/kubernetes/pkg/api/types.go#L1576